changes/7223.security

```diff
@@ -0,0 +1 @@
+Updated dependency `h11` to `0.16.0` to address `CVE-2025-43859`.
```

Comment:
Probably should note that:
1) this is an indirect dependency, so it won't autoupdate on upgrading Nautobot
2) if I'm reading poetry.lock correctly, `h11` is only a dependency of `wsproto` which is only a dependency of `trio-websocket` which is only a dependency of `selenium` so this *should* only impact dev environments.